presto-docs/src/main/sphinx/connector/kafka.rst

```diff
@@ -259,6 +264,430 @@ Field           Required  Type      Description
 
 There is no limit on field descriptions for either key or message.
 
+Kafka inserts
```

Comment:
Make these uppercase "Kafka Inserts" to match the existing titles in the document. While the new style guide says to use lowercase, that should only be followed for new documents, as we don't want to be inconsistent in the same document.